packages/main/src/MenuItem.ts

```diff
@@ -201,6 +207,54 @@ class MenuItem extends ListItem implements IMenuItem {
 	@i18n("@ui5/webcomponents")
 	static i18nBundle: I18nBundle;
 
+	_itemNavigation: ItemNavigation;
+	_lastFocusedItemIndex: number | null;
+
+	constructor() {
+		super();
+
+		this._itemNavigation = new ItemNavigation(this, {
+			getItemsCallback: () => this._navigableItems,
+		});
+		this._lastFocusedItemIndex = null;
+	}
+
+	get _navigableItems() {
+		let navigableItems: Array<HTMLElement> = [];
```

Comment:
You could directly return by using the spread operator on array initialization:
`return [...this.endContent] as Array<HTMLElement>;`  as this.endContent would be an empty array and `[...[]]` resolves to `[]`.